test/parallel/xyk-pallet.API.accuracy.tx.fees.test.ts

```diff
@@ -91,19 +91,18 @@ test("xyk-pallet - Calculate required MGA fee - CreatePool", async () => {
       second_asset_amount
     )
     .paymentInfo(testUser1.keyRingPair, opt);
-  await signAndWaitTx(
-    api?.tx.xyk.createPool(
-      firstCurrency,
+  await (await getMangataInstance())
+    .createPool(
+      testUser1.keyRingPair,
+      firstCurrency.toString(),
       first_asset_amount,
-      secondCurrency,
+      secondCurrency.toString(),
       second_asset_amount
-    ),
-    testUser1.keyRingPair,
-    await (await getCurrentNonce(testUser1.keyRingPair.address)).toNumber()
-  ).then((result) => {
-    const eventResponse = getEventResultFromTxWait(result);
-    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
-  });
+    )
+    .then((result) => {
+      const eventResponse = getEventResultFromTxWait(result);
+      expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+    });
```

Comment:
I see that you are using **async/await** and also **then** for promise. Why is that ? Maybe to choose only one ?

Reply:
It's about validating that the operation succeeds. I like it like that because is not part of the test, only validate that the operation was ok. 
BUT, 
if you think that is important, I can do it in an `async` `await`